src/lib/pipeline/adt/chunk/index.js

```diff
@@ -105,6 +108,19 @@ class Chunk extends THREE.Mesh {
     return bit & this.holes;
   }
 
+  dispose() {
+    this.geometry.dispose();
+    this.material.dispose();
+
+    this.material.textures.forEach((texture) => {
+      TextureLoader.unload(texture.sourceFile);
+    });
+
+    this.material.alphaMaps.forEach((alphaMap) => {
+      alphaMap.dispose();
+    });
+  }
```

Comment:
I believe that's what I did with WMO disposal-- just forgot to tidy up ADTs and M2s.
https://github.com/wowserhq/wowser/pull/109/files#diff-bc6db118c3eeba663ac1add52d72fe01R257